x-pack/plugins/canvas/public/components/workpad_manager/workpad_manager.js

```diff
@@ -50,7 +50,7 @@ export const WorkpadManager = ({ onClose }) => {
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiModalHeader>
-      <EuiModalBody>
+      <EuiModalBody className="canvasHomeApp__modalBody">
```

Comment:
Think something custom is messing up this modal 

![image](https://user-images.githubusercontent.com/324519/58664605-d2f57700-82e3-11e9-95f2-7211dc7b66c3.png)